Decent.Minecraft.Client/Blocks/Fire.cs

```diff
@@ -2,11 +2,14 @@
 
 namespace Decent.Minecraft.Client.Blocks
 {
+    /// <summary>
+    /// <a href="http://minecraft.gamepedia.com/Fire">Gamepedia link</a>.
+    /// </summary>
     public class Fire : Block
     {
         public Fire(byte intensity) : base(BlockType.Fire)
         {
-            if (intensity < 0 || intensity > 15)
+            if (intensity > 15)
```

Comment:
No, the right fix is to make the intensity an `int`, like I have on other blocks that take a numeric parameter. I made that change so we don't have that many casts around the code. The benefit of using a byte is negligible, but the drawbacks are everywhere.